libraries/AP_RCTelemetry/AP_GHST_Telem.cpp

```diff
@@ -0,0 +1,385 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "AP_GHST_Telem.h"
+#include <AP_VideoTX/AP_VideoTX.h>
+#include <AP_HAL/utility/sparse-endian.h>
+#include <AP_BattMonitor/AP_BattMonitor.h>
+#include <AP_GPS/AP_GPS.h>
+#include <GCS_MAVLink/GCS.h>
+#include <AP_RCProtocol/AP_RCProtocol_GHST.h>
+#include <AP_SerialManager/AP_SerialManager.h>
+#include <AP_Compass/AP_Compass.h>
+#include <AP_Baro/AP_Baro.h>
+#include <AP_AHRS/AP_AHRS.h>
+#include <AP_Notify/AP_Notify.h>
+#include <math.h>
+#include <stdio.h>
+#include <AP_HAL/AP_HAL.h>
+
+#if HAL_GHST_TELEM_ENABLED
```

Comment:
```suggestion
#if AP_GHST_TELEM_ENABLED
```

... and should be at top of file with only the `_config.h` include

Reply:
Fixed thanks